Rakefile

```diff
@@ -56,6 +57,17 @@ task :run, :command do |t, args|
   run_command args[:command]
 end
 
+desc "Updates the rspec.github.io docs"
+task :update_docs, [:version, :branch, :website_path] do |t, args|
+  args.with_defaults(:website_path => "../rspec.github.io")
```

Comment:
I remember before you had the website checked out as `rspec-website`.  You can pass a 3rd arg to set your path, but I figured `rspec.github.io` (what I have it cloned as) is more typical since that's the project name on github.  Or you can just rename your `rspec-website` directory :).